source/platforms/google-colab.md

```diff
@@ -0,0 +1,44 @@
+# RAPIDS on Google CoLab
+
+## 1. Set the Runtime
+
+In [Google Colab](https://colab.research.google.com/), click the `Runtime` dropdown at the top toolbar, then `Change Runtime Type`
+
+![Screenshot of create runtime and runtime type](../../images/googlecolab-select-runtime-type.png)
+
+Choose GPU for Hardware Accelerator
+
+![Screenshot of gpu for hardware accelerator](../../images/googlecolab-select-gpu-hardware-accelerator.png)
+
+## 2. Check GPU type
+
+Check the output of `!nvidia-smi` to make sure you've been allocated a Rapids Compatible GPU, i.e [Tesla T4, P4, or P100].
+
+![Screenshot of nvidia-smi](../../images/googlecolab-output-nvidia-smi.png)
+
+## 3. Run RAPIDS install script
+
+Checks to make sure that the GPU is RAPIDS compatible, Installs the **current stable** version of RAPIDSAI's core libraries using conda, which are: cuDF, cuML, cuGraph and xgboost.
+Please use the [RAPIDS Conda Colab Template notebook](https://colab.research.google.com/drive/1TAAi_szMfWqRfHVfjGSqnGVLr_ztzUM9) if you need to install any RAPIDS Extended libraries OR nightly version.
+
+```console
+# This gets the RAPIDS-Colab install files and test checks your GPU.
+# Please read the output of this cell.  If your Colab Instance is not RAPIDS compatible, it will warn you and give you remediation steps.
+
+!git clone https://github.com/rapidsai/rapidsai-csp-utils.git
+!python rapidsai-csp-utils/colab/install_rapids.py
```

Comment:
```suggestion
!python rapidsai-csp-utils/colab/install_rapids.py stable
```

Reply:
Running this even with `stable` on the end gives me the following error.

```
Installing RAPIDS Stable 22.12
Starting the RAPIDS install on Colab.  This will take about 15 minutes.
/bin/sh: 1: conda: not found
RAPIDS conda installation complete.  Updating Colab's libraries...
```

Reply:
cc @taureandyernv

Reply:
@taureandyernv  jw Can we install RAPIDS in CoLab via a brief standalone initialization script  (ie to avoid cloning this [github repo](https://github.com/rapidsai-community/rapidsai-csp-utils)) or via docker container?

Reply:
we'd REALLY rather not, as colab does things to break the install without warning, and users should have to deal with if they have the latest update.  It gets pretty untenable, as some people go months or years between uses, or sometimes the latest version of RAPIDS isn't compatible.  The abstraction of the script into a "nearly" always working template has provided us with the least amount of headaches so far.

Reply:
@taureandyernv thanks for your response! Can you also please address the traceback error above ? 

> Running this even with `stable` on the end gives me the following error.
> 
> ```
> Installing RAPIDS Stable 22.12
> Starting the RAPIDS install on Colab.  This will take about 15 minutes.
> /bin/sh: 1: conda: not found
> RAPIDS conda installation complete.  Updating Colab's libraries...
> ```